www/md_base/src/app/common/directives/panel/panel.directive.coffee

```diff
@@ -8,10 +8,7 @@ class Panel extends Directive
             controllerAs: 'panel'
             bindToController: true
             scope:
-                name: '='
-                title: '='
-                isCollapsed: '='
-                template: '='
+                bind: '='
```

Comment:
I better like the old interface, because it's more convenient. In this case you have to know what fields 'bind' should have. If you would like to keep this way, please put a comment somewhere near to 'scope' or 'bind' about the fields.